tutorials/custom-migration-tutorial.md

```diff
@@ -0,0 +1,109 @@
+--session-arg "collection_name:string='[COLLECTION_NAME]'" \
+--session-arg "hash_key_name:string='[ARG_HASH_KEY_NAME_1_0_0]'" \
+--session-arg "access_key_name:string='[ARG_ACCESS_KEY_NAME_1_0_0]'" 
+```
+
+Here is the full list of required arguments:
+- `node-address`: An IP address of a peer on the network. The default port for JSON-RPC servers on Mainnet and Testnet is 7777.
+- `chain-name`: The chain name of the network where you wish to send the deploy. For Mainnet, use *casper*. For Testnet, use *casper-test*.
+- `secret-key`: The file name containing the secret key of the account paying for the deploy.
+- `payment-amount`: The payment for the deploy in motes.
+- `session-path`: The path to the compiled Wasm on your computer. When using the [cep-78-wasm.tar.gz](https://github.com/casper-ecosystem/cep-78-enhanced-nft/releases/download/v1.1.1/cep-78-wasm.tar.gz) provided, this would be the path to the `contract.wasm` file.
+- `named_key_convention`: Argument that specifies the use of the `V_1_0_standard` [NamedKeyConvention](../README.md#namedkeyconventionmode).
+- `collection_name`: Argument that specifies the collection name as a String.
+- `hash_key_name`: The custom contract package hash NamedKey as a String.
+- `access_key_name`: The custom contract package access NamedKey as a String.
+
+The command returns the deploy hash that you can use to verify whether or not the deploy succeeded.
+
+**Important Notes**: 
+
+- Suppose an account attempts to install a second CEP-78 contract instance with the same collection name, but a custom contract package access NamedKey is used. In this case, the installation will not overwrite the NamedKey entry under which the access URef is written.
+- When installing a new version of the CEP-78 contract, you do not need to specify all the runtime arguments needed during the initial installation of version 1.0.0, such as `total_token_supply`, `ownership_mode`, etc. 
+
+**Example command to upgrade to v1.1.1:**
+
+The following is an example of upgrading and migrating to version 1.1.1 of a previously installed NFT collection using version 1.0.0 and with custom NamedKeys.
+
+```bash
+casper-client put-deploy \
+--node-addres http://65.21.235.219:7777 \
+--chain-name "casper-test" \
+--secret-key ~/KEYS/secret_key.pem \
+--payment-amount 300000000000 \
+--session-path contract.wasm \
+--session-arg "named_key_convention:u8='2'" \
+--session-arg "collection_name:string='CEP-78-collection'" \
+--session-arg "hash_key_name:string='mangled_hash_key'" \
+--session-arg "access_key_name:string='mangled_access_key'"
+```
+
+**Other Examples:**
+
+- A [Testnet account](https://testnet.cspr.live/account/013060d19fa5d5e471c2bbe88f35871454d2e162c444100eaca34671339c78ced4) that uses the custom NamedKeys "mangled_hash_key" and "mangled_access_key".
+- An example [Testnet deploy](https://testnet.cspr.live/deploy/59a785471600e183718b790fb19b3dec7242fde105928b9f90f01347b3c65f46) that specified the custom NamedKey convention.
```

Comment:
If I follow this link named_key_convention used was 1, not 2. It's a bit confusing.

Reply:
Thanks for catching this, I will updated it to https://testnet.cspr.live/deploy/55cb135c9b600263baedf72b124b02ff6dd74dd27d2d9be444b1bed6ee5e3301.